templates/pages/target-platforms/index.md

```diff
@@ -7,4 +7,46 @@ page_type: modular
 
 # Target platforms
 
+Ubuntu works on any distribution or target platform, from IoT devices to servers and desktops.
+
+{% include "includes/components/target-platforms.html" %}
+
+## Desktop
+
+Whether you are a beginner or a skilled developer, we recommend you install and test your software on the stable version of Ubuntu Desktop.
+
+[Download Ubuntu Desktop](https://www.ubuntu.com/download/desktop)
+
+For testers and early adopters, the daily desktop build version can be downloaded anytime.
+
+[Get the Ubuntu daily build](http://cdimage.ubuntu.com/daily-live/current/)
+
+With Ubuntu Make, you can easily install the latest stable version of the most popular IDEs, regardless of your language and technology of choice.
+
+[Install and use Ubuntu Make](#)
```

Comment:
Do we have a link for this? Otherwise I'm worried we may accidentally leave it without one.